actors/evm/tests/contracts/NotificationReceiver.sol

```diff
@@ -0,0 +1,627 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+// ========================================================
+// NOTE: If using this contract as an example, consider depending on utilities
+// available in https://github.com/filecoin-project/filecoin-solidity instead of
+// copying reusable utilities from here.
+// ========================================================
+
+contract NotificationReceiver {
+    // State variables to track received notifications
+    struct SectorNotification {
+        uint64 sector;
+        int64 minimumCommitmentEpoch;
+        bytes dataCid;
+        uint64 pieceSize;
+        bytes payload;
+    }
+    
+    SectorNotification[] public notifications;
+    mapping(uint64 => uint256[]) public sectorNotificationIndices;
+    
+    // Counter for total notifications received
+    uint256 public totalNotifications;
+    
+    // Flag to test different response behaviors
+    bool public shouldRejectNotifications = false;
+    
+    // Method selector for handle_filecoin_method
+    bytes4 constant NATIVE_METHOD_SELECTOR = 0x868e10c4;
+    
+    // Sector content changed method number
+    uint64 constant SECTOR_CONTENT_CHANGED = 2034386435;
+    
+    /**
+     * @dev Get the count of notifications for a specific sector
```

Comment:
`sol!` is picking up the javadoc style `/**` comments and `@dev` is being thrown into the mix when the macro is expanded; if you just replace all these comments with `//` then the tests pass I think

Reply:
Thanks, failure to check AI on my part